src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Status.java

```diff
@@ -6,6 +6,7 @@
 
     CONSUMED, // client service handled the documents
     CREATED,
+    FAILURE, // when we are aware of envelope, but there are inconsistency among files and metadata info
```

Comment:
We already have `UPLOAD_FAILURE`, `FAILURE` sounds very generic...
Perhaps something like 'metadata failure'???